immuni_common/models/mongoengine/batch_file.py

```diff
@@ -41,6 +41,10 @@ class BatchFile(Document):
     sub_batch_index: int = IntField()
     sub_batch_count: int = IntField()
 
+    origin: str = StringField(required=True)
+
+    batch_tag: str = StringField(null=True)
```

Comment:
Where can I find documentation on these fields?

Also, why do you need `null=True` instead of `required=False`?

Reply:
because the client springboot that communicates with European gateway need that field to be set to "null".

Reply:
I'd then add a `NOTE` comment. 🙏